packages/acala-oracle-dispatcher/PriceFetcher.ts

```diff
@@ -0,0 +1,43 @@
+import { CombinedFetcher, CCXTFetcher, CryptoCompareFetcher, FetcherInterface } from '@orml/fetcher';
+import config from './config';
+
+const CURRENCIES: { [key: string]: string } = {
+  BTC: 'XBTC',
+  ETH: 'ETH'
+};
+
+export default class PriceFetcher {
+  private readonly fetchers: { [key: string]: FetcherInterface };
+  private readonly symbols: string[];
+
+  constructor() {
+    this.symbols = config.symbols;
+
+    this.fetchers = this.symbols
+      .map((symbol) => {
+        const fetchers = config.exchanges[symbol]
+          .map((exchange) => [
+            new CCXTFetcher(exchange),
```

Comment:
no need to use CCXT and CryptoCompare to fetch a same exchange. We only want aggregated data from CryptoCompare, for exchange data we read it using CCXT directly.